### DISCUSSION TITLE
HashiCorp Vault Username Password Auth not supported in vault operator?

### DISCUSSION THREAD

**Discussion by quartelh** [bank-vaults/bank-vaults#2303] [Q&A] [answered]
Hi ,

according to the docs on https://developer.hashicorp.com/vault/docs/auth/userpass Username and Password is a authentication method.
We already implemented this approach in our java services.
We are currently looking into migrating to kubernetes. Both the java services and the hashicorp vaults will be moved into the cluster.

I have not found the username password option in the auth options of the vault operator ( according to https://bank-vaults.dev/docs/external-configuration/authentication/ ).

Am I missing something? 

Best regards,
Herbert

**Comment by akijakya** [comment, marked as answer]
Hi @quartelh, thanks for trying out Bank-Vaults! You can also configure the `userpass` auth method via the Operator, an example can be found [here](https://github.com/bank-vaults/bank-vaults/blob/e526dd770c706b124e803c8efb9a9c27f2f8a91d/vault-config.yml#L177). And we will definitely update the docs page regarding the auth methods!

**Reply by quartelh** [reply]
Thank you very much for pointing me into the right direction. Updating the documentation would really help (not for me , but for other users)
